api/v2.0/swagger.yaml

```diff
@@ -5334,6 +5334,99 @@ paths:
         '500':
           $ref: '#/responses/500'
 
+  /export/scan:
+    post:
+      summary: Export scan data for selected projects
+      description: Export scan data for selected projects
+      tags:
+        - scan data export
+      operationId: exportScanData
+      parameters:
+        - $ref: '#/parameters/requestId'
+        - $ref: '#/parameters/scanDataType'
+        - name: criteria
+          in: body
+          description: The criteria for the export
+          required: true
+          schema:
+            $ref: '#/definitions/ScanDataExportCriteria'
+      responses:
+        '200':
+          description: Success.
+          schema:
+            $ref: '#/definitions/ScanDataExportJob'
+        '400':
+          $ref: '#/responses/400'
+        '401':
+          $ref: '#/responses/401'
+        '403':
+          $ref: '#/responses/403'
+        '404':
+          $ref: '#/responses/404'
+        '405':
+          $ref: '#/responses/405'
+        '409':
+          $ref: '#/responses/409'
+        '500':
+          $ref: '#/responses/500'
+  /export/scan/execution/{execution_id}:
```

Comment:
Should change it  to a  list API, "/export/scan/executions", to list the executions for the current user

Reply:
And  the response of this API needs to carry the job name info(need to discuss how to implement)